soc/nxp/imxrt/rt5xx/Kconfig.soc

```diff
@@ -0,0 +1,60 @@
+# i.MX RT5XX Series
+
+# Copyright 2022-2024, NXP
+# SPDX-License-Identifier: Apache-2.0
+
+config SOC_SERIES_IMXRT_5XX
+	bool
+	select SOC_FAMILY_IMXRT
+
+config SOC_MIMXRT595S
+	bool
+	select SOC_SERIES_IMXRT_5XX
+
+config SOC_MIMXRT595S_CM33
+	bool
+	select SOC_MIMXRT595S
+	help
+	  NXP RT5xx CM33 core
+
+config SOC_MIMXRT595S_F1
+	bool
+	select SOC_MIMXRT595S
+	help
+	  NXP RT5xx ADSP Series
+
+config SOC_SERIES
+	default "rt5xx" if SOC_SERIES_IMXRT_5XX
```

Comment:
```suggestion
	default "imxrt_5xx" if SOC_SERIES_IMXRT_5XX
```

Reply:
Changed `CONFIG_SOC_SERIES` to `imxrt5xx` to align with rest of iMX RT series, and updated Kconfig name to `SOC_SERIES_IMXRT5XX`